docs/examples/upload_video.md

```diff
@@ -39,13 +34,13 @@ try {
 }
 
 echo 'Video ID: ' . $response['video_id'];
-~~~~
+```
 
-See more about the [`uploadVideo()` method](/docs/php/Facebook#upload-video).
+See more about the [`uploadVideo()` method](..reference/Facebook.md#uploadvideo).
```

Comment:
Broken link, should be `../reference/Facebook.md#uploadvideo` (missing slash). Can you check all link also please ? :)